lib/url-pattern/url-match-pattern-list.js

```diff
@@ -0,0 +1,88 @@
+var _ = require('../util').lodash,
+    PropertyList = require('../collection/property-list').PropertyList,
+    Url = require('../collection/url').Url,
+    UrlMatchPattern = require('./url-match-pattern').UrlMatchPattern,
+
+    MATCH_ALL_URLS = UrlMatchPattern.MATCH_ALL_URLS,
+
+    UrlMatchPatternList;
+
+_.inherit((
+  UrlMatchPatternList = function (parent, list) {
+      UrlMatchPatternList.super_.call(this, UrlMatchPattern, parent, list);
+  }), PropertyList);
+
+_.assign(UrlMatchPatternList.prototype, /** @lends UrlMatchPatternList.prototype */ {
+    toJSON: function() {
```

Comment:
Yeah, I mistook `PropertyList.prototype.toJSON` was not calling `member.toJSON()`. Turns out it does.

Removed this now.